File: 3731-find-missing-elements/3731-find-missing-elements.java
class Solution {
    public List<Integer> findMissingElements(int[] nums) {
        List<Integer> res=new ArrayList<>();
        int minn=nums[0];
        int maxx=nums[0];
        for(int i:nums)
        {
            minn=Math.min(minn,i);
            maxx=Math.max(maxx,i);
        }
        for(int i=minn;i<maxx;i++)
        {
            if(!contains(nums,i))
            {
                res.add(i);
            }
        }
        return res;
    }
    private boolean contains(int[] nums,int target)
    {
        for(int i:nums)
        {
            if(i==target)
            {
                return true;
            }
        }
        return false;
    }
}

File: 3731-find-missing-elements/3731-find-missing-elements.py
class Solution:
    def findMissingElements(self, nums: List[int]) -> List[int]:
        res=[]
        minn=min(nums)
        maxx=max(nums)
        for i in range(minn,maxx):
            if(i not in nums):
                res.append(i)
        return res